demo.py

```diff
@@ -1,2 +1,3 @@
 print('hello')
 
+print('what do you think about it??')
```

Comment:
make statement more specific 

```suggestion
print('what do you think about this statement?')
```